src/array/config.h

```diff
@@ -0,0 +1,34 @@
+/*!
```

Comment:
Move this to `include/dgl/runtime`?  Other C++ global configs can be put here as well.

Reply:
Sounds ok, and let me change the description from scope aten only to runtime.